### ISSUE TITLE
Use the new page features to create several pages instead of layers

### ISSUE THREAD

**Issue by brunnerant** [lifelike/countersheetsextension#99]
Hello,

In Inkscape 1.2, support for pages was added, so I was wondering if you planned to use that feature to put counters on several pages instead of several layers ? That way PDF generation would also be greatly simplified since we would be able to export all the pages as a single PDF file from Inkscape.

If not, I am happy to help develop that feature !

Thanks for your help and best regards !

**Comment by lifelike** [comment]
Yes, this would be good. A PR would be welcome. :) Otherwise I might get around to do it eventually, but no plans at the moment.
